.. mchoice:: itUpdate_MC_y
    :author: bmiller
    :difficulty: 3.0
    :basecourse: py4e-int
    :chapter: 05-iterations
    :subchapter: 05-updating
    :topics: 05-iterations/05-updating
    :from_source: T
    :practice: T
    :answer_a: The integer "2" prints.
    :answer_b: We get a TypeError.
    :answer_c: We get a NameError.
    :answer_d: The program compiles with no errors but nothing prints.
    :correct: c
    :feedback_a: Take another look at the second line.
    :feedback_b: This will not cause a TypeError because x and y are both integers.
    :feedback_c: This will cause a NameError because x has not been initialized yet.
    :feedback_d: This program will not compile.

    Consider the code block below. What happens when you run this program?

    .. code-block:: python

        y = 1
        x = x + 1
        print(x)